resources/lang/en/auth.php

```diff
@@ -5,4 +5,7 @@
 return [
     "failed" => "These credentials do not match our records.",
     "throttle" => "Too many login attempts. Please try again in :seconds seconds.",
+    "credentialsMismatch" => "Provided credentials do not match our records.",
```

Comment:
I'm pretty sure that common convention is to user snake_case for translations indexes: https://github.com/alexeymezenin/laravel-best-practices#follow-laravel-naming-conventions